internal/pkg/scaffold/cmd.go

```diff
@@ -150,9 +150,9 @@ func main() {
 	}
 
 	if err = serveCRMetrics(cfg); err != nil {
-		log.Info("Could not generate and serve custom resource metrics: ", err.Error())
+		log.Info("Could not generate and serve custom resource metrics", "error:", err.Error())
```

Comment:
Log message keys don't need colons. "error:" -> "error"

Reply:
@estroz thanks for the tip! I've fixed that now, and rebased.